performance-benchmarks/script

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+export BAL_HOME=$1
+chmod +x ${BAL_HOME}/bin/ballerina
```

Comment:
Why do you need this?

Reply:
Otherwise when running ballerina from script, we will get permission denied error

Reply:
It's already an executable file.